### PR TITLE
feat(codegen): INFRA-538 support fields params for loadListByChunks util

### PR DESCRIPTION
**Pull request by ekabardinsky**

### CODE REVIEW

File: packages/codegen/generate.server.utils.js [hunk 1]
```diff
@@ -641,6 +640,7 @@ function generateServerUtils (gqlOrSchemaName) {
         delete: delete_,
         softDelete,
         softDeleteMany,
+        hasFieldsParam: true, // TODO INFRA-538 remove this once migration to new server utils to be done
```
Comment (Alllex202) [inline packages/codegen/generate.server.utils.js:643]: What is this for?

Comment (Alllex202) [reply packages/codegen/generate.server.utils.js:643]: Ah, I see
Old utilities don't have this flag

Please write a comment about it

Comment (ekabardinsky) [reply packages/codegen/generate.server.utils.js:643]: This a temporary flag, just to notify developers - we have not migrated loadListByChunks utility to the new codegen utils.

File: apps/condo/domains/common/utils/serverSchema/index.js [hunk 2]
```diff
@@ -234,7 +236,16 @@ const loadListByChunks = async ({
             throw new Error('Operation timed out')
         }
 
-        newChunk = await list.getAll(context, where, { sortBy, first: chunkSize, skip: skip })
+        // TODO INFRA-538 remove this condition once migration to new server utils to be done
+        const resolvedFields = !isNil(fields) ? fields : 'id'
+        if (get(list, 'hasFieldsParam') && isNil(fields)) {
+            // the case when we migrate model to use new server utils, but loadListByChunks use it in old way
+            console.trace('Provide required fields parameter for loadListByChunks util')
+        }
```
Comment (Alllex202) [inline apps/condo/domains/common/utils/serverSchema/index.js:244]: Do you want the fields to be explicitly passed?
Isn't it enough that there is a default value ("id")?

Comment (ekabardinsky) [reply apps/condo/domains/common/utils/serverSchema/index.js:244]: Yes, 99.9% usages of loadListByChunks - get model data, not only ids. There are another case: model was migrated to the new codegen utils, but loadListByChunks are calling in old way. So I would like to notify developer about such case - to trigger him to rewrite code and provide required "fields" param.

**Review by AleX83Xpert — APPROVED** [review]

**Review by SavelevMatthew — APPROVED** [review]

**Review by abshnko — APPROVED** [review]

**Pull request merged**